java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/JdbcToArrowUtils.java

```diff
@@ -0,0 +1,343 @@
+                    case Types.BIT:
+                        BitVector bitVector = (BitVector) root.getVector(columnName);
+                        bitVector.setSafe(rowCount, rs.getBoolean(i)? 1: 0);
+                        bitVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.TINYINT:
+                        TinyIntVector tinyIntVector = (TinyIntVector)root.getVector(columnName);
+                        tinyIntVector.setSafe(rowCount, rs.getInt(i));
+                        tinyIntVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.SMALLINT:
+                        SmallIntVector smallIntVector = (SmallIntVector)root.getVector(columnName);
+                        smallIntVector.setSafe(rowCount, rs.getInt(i));
+                        smallIntVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.INTEGER:
+                        IntVector intVector = (IntVector)root.getVector(columnName);
+                        intVector.setSafe(rowCount, rs.getInt(i));
+                        intVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.BIGINT:
+                        BigIntVector bigIntVector = (BigIntVector)root.getVector(columnName);
+                        bigIntVector.setSafe(rowCount, rs.getInt(i));
+                        bigIntVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.NUMERIC:
+                    case Types.DECIMAL:
+                        DecimalVector decimalVector = (DecimalVector)root.getVector(columnName);
+                        decimalVector.setSafe(rowCount, rs.getBigDecimal(i));
+                        decimalVector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.REAL:
+                    case Types.FLOAT:
+                        Float4Vector float4Vector = (Float4Vector)root.getVector(columnName);
+                        float4Vector.setSafe(rowCount, rs.getFloat(i));
+                        float4Vector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.DOUBLE:
+                        Float8Vector float8Vector = (Float8Vector)root.getVector(columnName);
+                        float8Vector.setSafe(rowCount, rs.getDouble(i));
+                        float8Vector.setValueCount(rowCount + 1);
+                        break;
+                    case Types.CHAR:
+                    case Types.NCHAR:
+                    case Types.VARCHAR:
+                    case Types.NVARCHAR:
+                    case Types.LONGVARCHAR:
+                    case Types.LONGNVARCHAR:
+                        VarCharVector varcharVector = (VarCharVector)root.getVector(columnName);
+                        String value = rs.getString(i) != null ? rs.getString(i) : "";
```

Comment:
why a null value would be replaced by an empty string?

Reply:
I am not sure how I can set a null inside the VarCharVector. There is no API to set a NULL. Can you please suggest how this can be done?